source/_integrations/unifi.markdown

```diff
@@ -39,7 +39,7 @@ The UniFi controller allows you to create multiple users on it besides the main
 
 ### UniFi OS
 
-For UniFi OS a local-only user needs to be created. A user who uses the Ubiquiti cloud will not work. You can do this in the manage users section on the UniFi OS dashboard. Make sure to give it the right permissions for the functions you want to use. 
+For UniFi OS a local-only user needs to be created. A user who uses the Ubiquiti cloud will not work. You can do this in the manage users section on the UniFi OS dashboard. Make sure to give it the right permissions for the functions you want to use. Note the Dream Machine Pro needs the port to be 443. 
```

Comment:
This is a bit of a weird place to make a note about the dream machine?

Isn't the part of the configuration instructions?

Reply:
Sorry if this is the wrong place but the  Dream Machine pro (that runs UniFi OS) needs the port to be 443 as the default does not work. I did not see this anywhere in the configuration instructions.  I spent over an hour thinking I had the wrong user/password until I found a mention in the forum about the port.

Reply:
✅ Thanks 👍